.circleci/config.yml

```
@@ -35,6 +35,55 @@ references:
           POSTGRES_PASSWORD: mirror_node_pass
           POSTGRES_USER: mirror_node
 
+commands:
+  gcloud_image_untag:
+    description: "Untags old master images"
+    parameters:
+      imagepath:
+        type: string
+        default: "importer"
+    steps:
+      - run:
+          name: "Untags old master images"
+          command: |
+            set -x
+            UNTAG_BEFORE=$(date -d "-3 hours" '+%FT%T')
+            IMAGE_PATH=gcr.io/mirrornode/hedera-mirror-<< parameters.imagepath >>
+            OLD_IMAGES=$(gcloud container images list-tags ${IMAGE_PATH} \
+              --limit=1000 \
+              --sort-by=TIMESTAMP \
+              --filter="tags:image-* AND timestamp.datetime < '${UNTAG_BEFORE}'" \
+              --format="csv[no-heading](digest,tags,timestamp)")
+            for image in ${OLD_IMAGES[*]}; do
+            (
+              IFS=, read digest tag timestamp \<<< ${image}
+              gcloud container images untag -q "${IMAGE_PATH}:${tag}"
+            )
+            done
+  gcloud_image_delete:
+    description: "Deletes old untagged images"
+    parameters:
+      imagepath:
+        type: string
+        default: "importer"
+    steps:
+      - run:
+          name: "Deletes old untagged images"
+          command: |
+            set -x
```

Should be set -ex to fail fast